src/conversions.jl

```diff
@@ -140,9 +140,10 @@ pyptr_query(po::PyObject) = pyisinstance(po, c_void_p_Type) || pyisinstance(po,
 # typealias PyAny Union{PyObject, Int, Bool, Float64, Complex128, AbstractString, Function, Dict, Tuple, Array}
 abstract PyAny
 
-pyany_toany(T::Type) = T
-pyany_toany(T::Type{PyAny}) = Any
-pyany_toany(T::Type{Vararg{PyAny}}) = Vararg{Any}
+function pyany_toany(T::Type)
+    T === Vararg{PyAny} ? Vararg{Any} : T
+end
+pyany_toany(::Type{PyAny}) = Any
```

Comment:
I don't know the full story but it was never an actual type (you can't find a value that matches this type)